specification/iotcentral/resource-manager/Microsoft.IoTCentral/stable/2018-09-01/iotcentral.json

```diff
@@ -410,6 +410,43 @@
         "deprecated": false
       }
     },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.IoTCentral/appTemplates": {
```

Comment:
>appTemplates [](start = 68, length = 12)

Do app templates have any relationship with ARM templates? Also is their official name "App Templates" or "Application Templates"? If it's the latter, we should expand the name in the API.